collectors/0/iostat.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2.6
```

Comment:
Yes as python path depends on the OS (for example, FreeBSD : /usr/local/bin/python).
So for script which can be used on several OS, `/usr/bin/env python` is required.
So could be a best-practice to have it everywhere 👍